operator/v1/register.go

```diff
@@ -38,6 +38,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&KubeControllerManagerList{},
 		&OpenShiftAPIServer{},
 		&OpenShiftAPIServerList{},
+		&OpenShiftControllerManagerConfig{},
+		&OpenShiftControllerManagerListConfig{},
```

Comment:
@deads2k i have a feeling you're going to suggest dropping the Config suffix here(since you don't have it on your types), so i'm going to say up front that i think the suffix lends clarity and i'd prefer to keep it for our operator.